autogpt_platform/backend/backend/data/graph.py

```diff
@@ -596,15 +599,25 @@ async def get_graphs_executions(user_id: str) -> list[GraphExecutionMeta]:
         where={"userId": user_id},
         order={"createdAt": "desc"},
     )
-    return [GraphExecutionMeta.from_db(execution) for execution in executions]
+    return [
+        GraphExecutionMeta.from_db(
+            execution, await get_graph_execution_cost(execution.id)
```

Comment:
I'm not sure how much extra % of time/resources it costs to do this on every `GraphExecutionMeta` fetch. It would probably be a lot more efficient to do this through a `JOIN`, but we chose against that earlier. cc @majdyz